build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.App" Version="$(MicrosoftNETCoreAppPackageVersion)" />
+    <PackageReference Include="Microsoft.NETCore.App" Version="2.1.0-preview1-26116-04" />
```

Comment:
I'm surprised to see so many inline package versions here--even when these are static, shouldn't they still be centralized in DependencyVersions.props like VersionTools and the Feed package?

Reply:
Yeah I agree with @dagood, but I think that doesn't block this check-in (since there are a bunch of other non-centralized properties here).